utilities/pipelines/resourceRemoval/Remove-DeployedModuleWithChildResources.ps1

```diff
@@ -0,0 +1,145 @@
+﻿<#
```

Comment:
Will integrate this code into the removal script via a dedicated PR into this branch.

Reply:
Started in branch `users/alsehr/removalImprovement_export`